.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,19 @@
+name: Issue Submission to Pull Request
+on:
+  issues:
```

Comment:
Should this not be a check on PR's where any `./.github/scripts/*.sh` files are edited?